business-registry-dashboard/app/components/Table/Column/Header.vue

```diff
@@ -11,9 +11,9 @@ defineProps<{
 defineEmits(['clear'])
 </script>
 <template>
-  <div class="flex flex-col gap-2 font-normal">
+  <div class="flex flex-col gap-2 border-b border-gray-200 font-normal">
```

Comment:
add border so theres a bottom border visible when scrolling the table, separating the thead and tbody